Assets/Managers/waterManager.cs

```diff
@@ -0,0 +1,76 @@
+using System.Collections;
```

Comment:
Happy with this system, I think it will expand well to the other systems (dirt, themes, etc.)